examples/braket_features/TN1_demo_local_vs_non-local_random_circuits.ipynb

```diff
@@ -0,0 +1,889 @@
+{
```

Comment:
<p>Nit: it's -&gt; its</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/aws/amazon-braket-examples/pull/47/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='9'/>

Reply:
Fixed.